### ISSUE TITLE
Main page view ordered by alphabetical order and not following PriceDrop order

### ISSUE THREAD

**Issue by RodrigoHornos** [PrestaShop/PrestaShop#18724]
#### Describe the bug

When using the module in the main page, the products are shown by alphabetical order, which is not useful. Also changing the order in the controller does not affect this view. 

#### Expected behavior

It is expected to have the same order view than the PriceDrop controller.

#### Steps to Reproduce

Setting 
0. change /controllers/front/listing/PricesDropController.php line 49 to: 
            ->setSortOrder(new SortOrder('product', 'price', 'asc'));
1. Go to Main Page
2. Check the product order in the ps_specials frame
3. Click on the "All sale products" to price drop page
4. Check different order

#### Additional information

* PrestaShop version: 1.7.6.2
* PHP version: 7.2

**Comment by sarahdib** [comment]
Hello @RodrigoHornos 

Thanks for your feedback !

@PrestaShop/prestashop-product-team what do you think about this improvement ? Change the sort by default of the listing product in modules ?

**Comment by colinegin** [comment]
This could be interesting ! Maybe not to change it for all but to give the possibility to adapt the configuration.
@LouiseBonnard do you think it could be a good first issue ?

**Comment by LouiseBonnard** [comment]
Something quite similar has been done for the Faceted Search module to be able to sort products by stock availability: https://github.com/PrestaShop/ps_facetedsearch/pull/51. But I guess this set up was more complex, it looks more simple here. @PrestaShop/prestashop-core-developers, what do you think?

**Comment by sam-pires** [comment]
To me it seems this is rather a bug, no?
The default behavior should not be modified (and stuck) when using the module.

**Comment by RodrigoHornos** [comment]
The bug is referred to the Pricedrop module for the main page.  There is no sense to show products in alphabetical order here. It should be by top discount % or by lower prices in order to boost sales from the main page.

**Comment by LouiseBonnard** [comment]
Indeed, by default, the order on the Specials Block dedicated page on the front office is by name (A to Z). But I can manage to modify the order with the dropdown list on the top right-hand corner and sort products by price.

So, as an improvement, we could add a drop-down in the module's configuration panel in the back office to let the users select the default order of their choice.

**Comment by marionf** [comment]
@LouiseBonnard 

Or instead of preselect a sort by name A to Z by default, we can add and preselect "Relevance" that is not displayed on prices-drop page currently. 
This way, it will work like on categories pages: "Relevance" is preselected and depends on what the merchant configured in the BO > Shop parameters > Products > default order by & default order method.

WDYT ?

![capture d'écran_2558](https://user-images.githubusercontent.com/13449658/80080416-e3dd5580-8551-11ea-9b8a-70cdcaafedec.png)

![capture d'écran_2559](https://user-images.githubusercontent.com/13449658/80080423-e6d84600-8551-11ea-852d-ecf88b9cc91b.png)
